keyboards/akko/5075/5075.c

```diff
@@ -0,0 +1,178 @@
+    {0, CB7_CA6,   CB8_CA6,   CB9_CA6},
+    {0, CB7_CA7,   CB8_CA7,   CB9_CA7},
+    {0, CB7_CA8,   CB8_CA8,   CB9_CA8},
+    {0, CB7_CA9,   CB8_CA9,   CB9_CA9},
+    {0, CB7_CA10,  CB8_CA10,  CB9_CA10},
+    {0, CB7_CA11,  CB8_CA11,  CB9_CA11},
+    {0, CB7_CA12,  CB8_CA12,  CB9_CA12},
+    {0, CB1_CA15,  CB2_CA15,  CB3_CA15},
+    {0, CB7_CA13,  CB8_CA13,  CB9_CA13},
+    {1, CB4_CA3,   CB5_CA3,   CB6_CA3},
+
+    {0, CB10_CA1,   CB11_CA1,   CB12_CA1},
+    {0, CB1_CA16,  CB2_CA16,  CB3_CA16},
+    {0, CB10_CA2,   CB11_CA2,   CB12_CA2},
+    {0, CB10_CA3,   CB11_CA3,   CB12_CA3},
+    {0, CB10_CA4,   CB11_CA4,   CB12_CA4},
+    {0, CB10_CA5,   CB11_CA5,   CB12_CA5},
+    {0, CB10_CA6,   CB11_CA6,   CB12_CA6},
+    {0, CB10_CA7,   CB11_CA7,   CB12_CA7},
+    {0, CB10_CA8,   CB11_CA8,   CB12_CA8},
+    {0, CB10_CA9,   CB11_CA9,   CB12_CA9},
+    {0, CB10_CA10,  CB11_CA10,  CB12_CA10},
+    {0, CB10_CA11,  CB11_CA11,  CB12_CA11},
+    {0, CB10_CA12,  CB11_CA12,  CB12_CA12},
+    {1, CB4_CA7,   CB5_CA7,   CB6_CA7},
+    {1, CB4_CA4,   CB5_CA4,   CB6_CA4},
+
+    {0, CB10_CA13,  CB11_CA13,  CB12_CA13},
+    {0, CB10_CA14,  CB11_CA14,  CB12_CA14},
+    {0, CB10_CA15,  CB11_CA15,  CB12_CA15},
+    {0, CB10_CA16,  CB11_CA16,  CB12_CA16},
+
+    {0, CB7_CA14,  CB8_CA14,  CB9_CA14},
+    {0, CB7_CA15,  CB8_CA15,  CB9_CA15},
+    {0, CB7_CA16,  CB8_CA16,  CB9_CA16},
+
+    {0, CB4_CA15,  CB5_CA15,  CB6_CA15},
+    {1, CB4_CA6,   CB5_CA6,   CB6_CA6},
+    {1, CB4_CA5,   CB5_CA5,   CB6_CA5},
+
+    {1, CB7_CA1,   CB8_CA1,   CB9_CA1},
+    {1, CB7_CA2,   CB8_CA2,   CB9_CA2},
+    {1, CB7_CA3,   CB8_CA3,   CB9_CA3},
+    {1, CB7_CA4,   CB8_CA4,   CB9_CA4},
+    {1, CB7_CA5,   CB8_CA5,   CB9_CA5},
+    {1, CB7_CA6,   CB8_CA6,   CB9_CA6},
+    {1, CB7_CA7,   CB8_CA7,   CB9_CA7},
+    {1, CB7_CA8,   CB8_CA8,   CB9_CA8},
+    {1, CB7_CA9,   CB8_CA9,   CB9_CA9},
+    {1, CB7_CA10,  CB8_CA10,  CB9_CA10},
+    {1, CB7_CA11,  CB8_CA11,  CB9_CA11},
+
+    {1, CB10_CA1,   CB11_CA1,   CB12_CA1},
+    {1, CB10_CA2,   CB11_CA2,   CB12_CA2},
+    {1, CB10_CA3,   CB11_CA3,   CB12_CA3},
+    {1, CB10_CA4,   CB11_CA4,   CB12_CA4},
+    {1, CB10_CA5,   CB11_CA5,   CB12_CA5},
+    {1, CB10_CA6,   CB11_CA6,   CB12_CA6},
+    {1, CB10_CA7,   CB11_CA7,   CB12_CA7},
+    {1, CB10_CA8,   CB11_CA8,   CB12_CA8},
+    {1, CB10_CA9,   CB11_CA9,   CB12_CA9},
+    {1, CB10_CA10,  CB11_CA10,  CB12_CA10},
+    {1, CB10_CA11,  CB11_CA11,  CB12_CA11},
+};
+
+#endif
+// clang-format on
+void keyboard_pre_init_kb(void) {
+    // set our LED pings as output
+    gpio_set_pin_output(LED_MAC_OS_PIN); // LDE2 MAC\WIN
+    gpio_write_pin_low(LED_MAC_OS_PIN);
+    gpio_set_pin_output(LED_WIN_LOCK_PIN); // LED3 Win Lock
+    gpio_write_pin_low(LED_WIN_LOCK_PIN);
+
+    housekeeping_task_kb();
+}
+
+void housekeeping_task_kb(void) {
+    gpio_write_pin(LED_MAC_OS_PIN, ((get_highest_layer(default_layer_state | layer_state) == 2) || (get_highest_layer(default_layer_state | layer_state) == 3)));
```

Comment:
This is making the LED that's supposed to inform if the keyboard is on Mac or Windows have a strange behaviour: it informs we are on the default layer on Mac by turning on, but turns off if the FN key is pressed. On Windows, it turns off on default layer as expected, but will actually turn on when we press the FN key. Maybe something like this would be better:
```suggestion
        gpio_write_pin(LED_MAC_OS_PIN, 
    (get_highest_layer(default_layer_state | layer_state) == 3) ||
    (get_highest_layer(default_layer_state | layer_state) == 4) ||
    (get_highest_layer(default_layer_state | layer_state) == 5));
```
This will only turn the led on when any layer related to Mac is active.